vars/tdr.groovy

```diff
@@ -42,6 +42,22 @@ def getAccountNumberFromStage(String stage) {
   return stageToAccountMap.get(stage)
 }
 
+def commitAndPushGitHubBranch(String branch, String commitMessage) {
+  sshagent(['github-jenkins']) {
+    sh "git config --global user.email tna-digital-archiving-jenkins@nationalarchives.gov.uk"
+    sh "git config --global user.name tna-digital-archiving-jenkins"
```

Comment:
Is it worth commonising this setup with the other tasks that use the Jenkins GitHub user, so we're not referencing this user in multiple places?

Reply:
Not sure I follow here. Both the "commit" and the push commands require the Github user. Are you suggesting, making this even more generic by passing in the git commands as a param? Not sure how that would work?

Reply:
I was thinking of adding a sort of setup function which configures git, e.g.

```
def configureJenkinsGitUser() {
  sh "git config --global user.email tna-digital-archiving-jenkins@nationalarchives.gov.uk"
  sh "git config --global user.name tna-digital-archiving-jenkins"
}
```

The job can call that once at the start, or when it's about to do some git wrangling.

Reply:
Changes made